src/fiskaltrust.Launcher/ServiceInstallation/LinuxSystemD.cs

```diff
@@ -19,17 +23,29 @@ public override async Task<int> InstallService(string commandArgs, string? displ
             {
                 return -1;
             }
+
+            // Creating a directory if does not exist
+            if (!Directory.Exists(_requiredDirectory))
+            {
+                Directory.CreateDirectory(_requiredDirectory);
+
+                // Change of directory owner
+                await RunProcess("chown", new[] { _serviceUser, _requiredDirectory });
+
+                // Changing directory permissions
+                await RunProcess("chmod", new[] { "700", _requiredDirectory });
+            }
+
             Log.Information("Installing service via systemd.");
             var serviceFileContent = GetServiceFileContent(displayName ?? "Service installation of fiskaltrust launcher.", commandArgs);
             var serviceFilePath = Path.Combine(_servicePath, $"{_serviceName}.service");
-            await File.AppendAllLinesAsync(serviceFilePath, serviceFileContent).ConfigureAwait(false);
+            await File.WriteAllTextAsync(serviceFilePath, string.Join("\n", serviceFileContent)).ConfigureAwait(false);
```

Comment:
Why the leading newline? 🤔

Reply:
I've added it because I thought it's standard what I already read and have seen in different solutions. In the implementation of `GetServiceFileContent` – there's no leading newline in the array elements - that's the reason why but imo this should not be a problem

Reply:
ah ok. I did not see we're now using the `*AllTextAsync` method instead of `*AllLinesAsync`.